### DISCUSSION TITLE
Nuxt3 Stable Version Release Date

### DISCUSSION THREAD

**Discussion by bahmanim** [nuxt/nuxt#16915] [General]
When will the stable version be released????
On the site, the release time was June, but no stable version has arrived yet.

**Comment by danielroe** [comment]
Please see https://v3.nuxtjs.org/community/roadmap/.

**Reply by bahmanim** [reply]
I have read this page , and the date of stable version for nuxt3 is Midsummer, 2022.
But Haven't we passed the middle of summer?

**Reply by lionel-coderm** [reply]
I'm also looking forward to the release

**Comment by kosmeln** [comment]
Hi @danielroe, @pi0.
The roadmap is outdated and still states Summer 2022 as a planned release date. Is there a plan of re-evaluating the estimated release date and/or the scope of work that should be complete before a go live?
I know there are a lot of folks around curious about it so it'd be great to provide some input here.

Thank you!

**Reply by danielroe** [reply]
The roadmap is still correct. https://www.google.com/search?q=summer+2022

**Reply by procrates** [reply]
Totally get that, in Norway the summer ends 23. September, is that the same as the UK. Would that be correct for the southern hemisphere too (just so this wont be a question again..)?
RIP Queen Elizabeth II

**Reply by danielroe** [reply]
Yes, that's what we're working with too... 😊
